Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1390,41 +1390,28 @@ sent from outside the cluster to the service, the request's source IP address is
 used for determining the endpoint affinity. If a request is sent from inside
 the cluster, then the source depends on whether the socket-LB feature
 is used to load balance ClusterIP services. If yes, then the client's network
-namespace cookie is used as the source. The latter was introduced in the 5.7
-Linux kernel to implement the affinity at the socket layer at which
-the socket-LB operates (a source IP is not available there, as the
-endpoint selection happens before a network packet has been built by the
+namespace cookie is used as the source - it allows to implement affinity at the
+socket layer at which the socket-LB operates (a source IP is not available there,
+as the endpoint selection happens before a network packet has been built by the
 kernel). If the socket-LB is not used (i.e. the loadbalancing is done
 at the pod network interface, on a per-packet basis), then the request's source
 IP address is used as the source.
 
-The session affinity support is enabled by default for Cilium's kube-proxy
-replacement. For users who run on older kernels which do not support the network
-namespace cookies, a fallback in-cluster mode is implemented, which is based on
-a fixed cookie value as a trade-off. This makes all applications on the host to
-select the same service endpoint for a given service with session affinity configured.
```

Comment:
@brb for the record, I'm not actually sure where the code for this "fixed cookie value" lives / lived. And whether it's all been removed :slightly_smiling_face: